src/Serilog.Sinks.AwsCloudWatch/Serilog.Sinks.AwsCloudWatch.csproj

```
@@ -28,7 +28,7 @@
     <None Include="../../readme.md" pack="true" PackagePath="." />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="[3.5.0.9,)" />
+    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.7.1" />
```

Updating to allow the range.
```suggestion
    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="[3.7.1,)" />
```